src/stories/molecules/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,35 @@
+// Global Imports
+import React, { ReactNode } from 'react';
+
+// Local Imports
+import { Button } from 'stories/atoms/Button/Button';
+// I know the caret has to wrapped in a wrapper so we can adjust it's placement and allow for roatation, but my first goal was just to render the Caret somewhere on the button first
+// import { DropCaretWrapper } from 'stories/molecules/Dropdown/Dropdown.style';
+// import { SvgContainer } from 'stories/atoms/SVGIcon/svgicon-styles';
+import { ReactComponent as CaretSvg } from 'stories/assets/caret_icon.svg';
+
+interface Props {
```

Comment:
Get in the habit of commenting all of your props. I like to phrase mine as a question:

```ts
interface Props {
/** What should the Dropdown inner text be? */
label: string;
}
```

Also, note the use of `/** */` for leaving a comment. That leaves it in `JSDocs` format. Read more about that [here](https://jsdoc.app/).